docs/tutorial/10-resources-compose.mdx

```diff
@@ -133,7 +133,7 @@ These definitions show how a Kitty resource could own hats.
 The hats are stored in a variable in the Kitty resource.
 
     // place where the Kitty hats are stored
```

Comment:
Wrap in a code fence instead:

```suggestion
```cadence
    // place where the Kitty hats are stored
```